client/src/components/Login/Login.js

```diff
@@ -1,7 +1,81 @@
 import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Grid from "@material-ui/core/Grid";
+import { Button, Container, Typography } from "@material-ui/core";
+import FacebookLogo from "./assets/FacebookLogo";
+import GoogleLogo from "./assets/GoogleLogo";
+
+const useStyles = makeStyles((theme) => ({
+  titleText: {
+    alignText: "center",
+  },
+  row: {
+    display: "flex",
+    alignItems: "center",
+    justifyContent: "center",
+  },
+  loginButton: {
+    backgroundColor: "#EEEEEE",
+    textTransform: "capitalize",
+    fontFamily: theme.fontFamily,
```

Comment:
I think we should make the buttons the same size

```suggestion
    fontFamily: theme.fontFamily,
    width: "200px"
```